cosmos/providers/dbt/core/operators_docker.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import annotations
+
+from typing import Sequence
+
+import yaml
+from airflow.providers.docker.operators.docker import DockerOperator
+from airflow.utils.context import Context
+
+from cosmos.providers.dbt.core.operators import DbtBaseOperator
+
+
+class DbtDockerBaseOperator(DockerOperator, DbtBaseOperator):
+    """
+    Executes a dbt core cli command in a Docker container.
+
+    """
+
+    template_fields: Sequence[str] = (
+        DbtBaseOperator.template_fields + DockerOperator.template_fields
+    )
+
+    def __init__(
+        self,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+
+    def build_cmd_and_args(self, env: dict = {}, cmd_flags: list = None):
+        dbt_cmd, env_vars = self.build_cmd(
+            env=env, cmd_flags=cmd_flags, handle_profile=False
+        )
+
+        ## set env vars
+        self.environment = {**env_vars, **self.environment}
```

Comment:
I think we can use this line to set the profile here to set the "DBT_PROFILES_DIR" env var.

Reply:
Done